tests/test_auth.py

```diff
@@ -11,10 +11,21 @@ def setUp(self) -> None:
         self.user = StaffUserFactory()
         self.url = reverse("admin:auth_user_export", kwargs={"object_id": self.user.id})
 
-    def test_get_request_export_mixin(self):
+    def test_export_mixin_redirect(self):
```

Comment:
It looks like this test checks the same thing as the test below

Reply:
Changed